app/models/article.rb

```diff
@@ -30,17 +32,18 @@ class Article < ActiveRecord::Base
   has_many :articles_courses, class_name: ArticlesCourses
   has_many :courses, -> { uniq }, through: :articles_courses
   has_many :assignments
+  belongs_to :wiki
+
+  # Allows us to directly #update from the Tools response
+  alias_attribute :page_id, :native_id
```

Comment:
Are you talking about being able to use the `page_id` key on both sides of the assignment operator [here](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/773d72ce4169dc78d93fa9cb5c63799406cd9a07/lib/replica.rb#L71)? It seems that a detail of how the importer works shouldn't be in the model... I'd remove the alias, and maybe just call the real attribute/column something other than `native_id` (and do the same for other columns being added) to make the code that uses the model clearer.